traffic_ops/traffic_ops_golang/ats/db.go

```diff
@@ -197,3 +346,428 @@ WHERE
+	qry := RemapDSDataQuerySelectFrom + RemapDSDataQueryWhereForMid + RemapDSDataQueryOrderBy
+	rows, err := tx.Query(qry, serverInfo.CDN)
+	if err != nil {
+		return nil, errors.New("querying: " + err.Error())
+	}
+	defer rows.Close()
+
+	dses := []atscfg.RemapConfigDSData{}
+	for rows.Next() {
+		d := atscfg.RemapConfigDSData{}
+		if err := rows.Scan(&d.Name, &d.ID, &d.DSCP, &d.RoutingName, &d.SigningAlgorithm, &d.QStringIgnore, &d.OriginFQDN, &d.MultiSiteOrigin, &d.RangeRequestHandling, &d.FQPacingRate, &d.OriginShield, &d.Pattern, &d.RegexType, &d.Type, &d.Domain, &d.RegexSetNumber, &d.EdgeHeaderRewrite, &d.MidHeaderRewrite, &d.RegexRemap, &d.CacheURL, &d.RemapText, &d.Protocol, &d.ProfileID, &d.AnonymousBlockingEnabled, &d.Active); err != nil {
+			return nil, errors.New("scanning: " + err.Error())
+		}
+		if !RemapDotConfigIncludeInactiveDeliveryServices && !d.Active {
+			continue
+		}
+		d.Type = tc.DSTypeFromString(string(d.Type))
+		dses = append(dses, d)
+	}
+	return dses, nil
+}
+
+func GetRemapDSDataForEdge(tx *sql.Tx, server *atscfg.ServerInfo) ([]atscfg.RemapConfigDSData, error) {
+	qry := RemapDSDataQuerySelectFrom + RemapDSDataQueryWhereForEdge + RemapDSDataQueryOrderBy
+	rows, err := tx.Query(qry, server.ID)
+	if err != nil {
+		return nil, errors.New("querying: " + err.Error())
+	}
+	defer rows.Close()
+
+	dses := []atscfg.RemapConfigDSData{}
+	for rows.Next() {
+		d := atscfg.RemapConfigDSData{}
+		if err := rows.Scan(&d.Name, &d.ID, &d.DSCP, &d.RoutingName, &d.SigningAlgorithm, &d.QStringIgnore, &d.OriginFQDN, &d.MultiSiteOrigin, &d.RangeRequestHandling, &d.FQPacingRate, &d.OriginShield, &d.Pattern, &d.RegexType, &d.Type, &d.Domain, &d.RegexSetNumber, &d.EdgeHeaderRewrite, &d.MidHeaderRewrite, &d.RegexRemap, &d.CacheURL, &d.RemapText, &d.Protocol, &d.ProfileID, &d.AnonymousBlockingEnabled, &d.Active); err != nil {
+			return nil, errors.New("scanning: " + err.Error())
+		}
+		if !RemapDotConfigIncludeInactiveDeliveryServices && !d.Active {
+			continue
+		}
+		d.Type = tc.DSTypeFromString(string(d.Type))
+		dses = append(dses, d)
+	}
+	return dses, nil
+}
+
+func GetServerNameFromID(tx *sql.Tx, id int) (tc.CacheName, bool, error) {
+	qry := `SELECT s.host_name FROM server s WHERE s.id = $1`
+	name := tc.CacheName("")
+	if err := tx.QueryRow(qry, id).Scan(&name); err != nil {
+		if err == sql.ErrNoRows {
+			return "", false, nil
+		}
+		return "", false, errors.New("querying: " + err.Error())
+	}
+	return name, true, nil
+}
+
+// getServerNameFromNameOrID returns the server name from a parameter which may be the name or ID.
+// This also checks and verifies the existence of the given server, and returns an appropriate user error if it doesn't exist.
+// Returns the name, any user error, any system error, and any error code.
+func GetServerNameFromNameOrID(tx *sql.Tx, serverNameOrID string) (tc.CacheName, error, error, int) {
```

Comment:
Capitalization of the name in the comment should match the name of the function.

Reply:
Not commenting everywhere, but there are a bunch of mismatches.

Reply:
Yeah, funcs I renamed and didn't fix the comment. I'll fix.

Reply:
Fixed.